descriptors/ModuleDescriptor-template.json

```diff
@@ -343,14 +357,40 @@
       "displayName" : "Get refreshed presigned url for export file",
       "description" : "Get refreshed presigned url for export file"
     },
+    {
+      "permissionName" : "permissions-self-check.get",
+      "displayName" : "Set of desired permissions",
+      "description" : "Set of desired permissions"
+    },
+    {
+      "permissionName" : "bulk-edit.users.get",
```

Comment:
these permissions should be declared in mod-bulk-ops

Reply:
hi @NikitaSedyx , it is read permission to retrieve entities by identifiers. This logic is located in mod-data-export-worker.